wasmbinding/queries_list.go

```diff
@@ -0,0 +1,80 @@
+	incentivestypes "github.com/osmosis-labs/osmosis/v13/x/incentives/types"
+	lockuptypes "github.com/osmosis-labs/osmosis/v13/x/lockup/types"
+	minttypes "github.com/osmosis-labs/osmosis/v13/x/mint/types"
+	poolincentivestypes "github.com/osmosis-labs/osmosis/v13/x/pool-incentives/types"
+	superfluidtypes "github.com/osmosis-labs/osmosis/v13/x/superfluid/types"
+	tokenfactorytypes "github.com/osmosis-labs/osmosis/v13/x/tokenfactory/types"
+	twapquerytypes "github.com/osmosis-labs/osmosis/v13/x/twap/client/queryproto"
+	twapv2querytypes "github.com/osmosis-labs/osmosis/v13/x/twap/client/v2queryproto"
+	txfeestypes "github.com/osmosis-labs/osmosis/v13/x/txfees/types"
+)
+
+var (
+	QueriesList = []codec.ProtoMarshaler{
+		&authtypes.QueryAccountResponse{},
+		&authtypes.QueryParamsResponse{},
+		&banktypes.QueryBalanceResponse{},
+		&banktypes.QueryDenomsMetadataResponse{},
+		&banktypes.QueryParamsResponse{},
+		&banktypes.QuerySupplyOfResponse{},
+		&distributiontypes.QueryParamsResponse{},
+		&distributiontypes.QueryDelegatorWithdrawAddressResponse{},
+		&distributiontypes.QueryValidatorCommissionResponse{},
+		&govtypes.QueryDepositResponse{},
+		&govtypes.QueryParamsResponse{},
+		&govtypes.QueryVoteResponse{},
+		&slashingtypes.QueryParamsResponse{},
+		&slashingtypes.QuerySigningInfoResponse{},
+		&stakingtypes.QueryDelegationResponse{},
+		&stakingtypes.QueryParamsResponse{},
+		&stakingtypes.QueryValidatorResponse{},
+		&epochtypes.QueryEpochsInfoResponse{},
+		&epochtypes.QueryCurrentEpochResponse{},
+		&gammtypes.QueryNumPoolsResponse{},
+		&gammtypes.QueryTotalLiquidityResponse{},
+		&gammtypes.QueryPoolResponse{},
+		&gammtypes.QueryPoolParamsResponse{},
+		&gammtypes.QueryTotalPoolLiquidityResponse{},
+		&gammtypes.QueryTotalSharesResponse{},
+		&gammtypes.QueryPoolTypeResponse{},
+		&gammv2types.QuerySpotPriceResponse{},
+		&gammtypes.QuerySwapExactAmountInResponse{},
+		&incentivestypes.ModuleToDistributeCoinsResponse{},
+		&incentivestypes.QueryLockableDurationsResponse{},
+		&lockuptypes.ModuleBalanceResponse{},
+		&lockuptypes.ModuleLockedAmountResponse{},
+		&lockuptypes.AccountUnlockableCoinsResponse{},
+		&lockuptypes.AccountUnlockingCoinsResponse{},
+		&lockuptypes.LockedDenomResponse{},
+		&minttypes.QueryEpochProvisionsResponse{},
+		&minttypes.QueryParamsResponse{},
+		&poolincentivestypes.QueryGaugeIdsResponse{},
+		&superfluidtypes.QueryParamsResponse{},
+		&superfluidtypes.AssetTypeResponse{},
+		&superfluidtypes.AllAssetsResponse{},
+		&superfluidtypes.AssetMultiplierResponse{},
+		&txfeestypes.QueryFeeTokensResponse{},
+		&txfeestypes.QueryDenomSpotPriceResponse{},
+		&txfeestypes.QueryDenomPoolIdResponse{},
+		&txfeestypes.QueryBaseDenomResponse{},
+		&tokenfactorytypes.QueryParamsResponse{},
+		&tokenfactorytypes.QueryDenomAuthorityMetadataResponse{},
+		&twapv2querytypes.ArithmeticTwapResponse{},
+		&twapv2querytypes.ArithmeticTwapToNowResponse{},
+		&twapquerytypes.ParamsResponse{},
```

Comment:
This is far more queries than we had before. Lets please do two different PR's, and leave the query list untouched in teh first one, beyond route removal.